packages/suite/src/utils/wallet/sendFormUtils.ts

```diff
@@ -260,7 +262,9 @@ export const getBitcoinComposeOutputs = (values: Partial<FormState>, symbol: Acc
     // corner case for multiple outputs
     // one Output is valid and "final" but other has only address
     // to prevent composing "final" transaction switch it to not-final (noaddress)
-    const hasIncompleteOutput = values.outputs.find(o => o && o.address && !o.amount);
+    const hasIncompleteOutput = values.outputs.find(
+        (o, i) => setMaxOutputId !== i && o && o.address && !o.amount,
```

Comment:
Minor thing here, could use optional chaining. And it's not very clear what the amount check is supposed to be. I believe amount is a number? Is it checking for a missing or 0 amount in this case?

Reply:
amount is a string (all fields from the inputs are strings, [typed here](https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/types/wallet/sendForm.ts#L19)) so in this case: amount === undefined (when input was not registered yet) or amount === '' (amount is empty)